wavelength-plugins/friends.js

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Normally, multiline comments using the `/** */` syntax have two asterisks (`*`) before the first newline.